### PR TITLE
Fix the newsletters JS and stop the 403

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

- Adds missing file to the lists in curlConfig and webpack
- Rename `signup.js` to `newsletters.js`, since this code only runs on the newsletters page...

## What is the value of this and can you measure success?

- Allow `newsletters.js` to load on https://www.theguardian.com/email-newsletters
- Newsletters page can be enhanced

## Does this affect other platforms - Amp, Apps, etc?

Nope

-------
@gtrufitt

### CODE REVIEW

File: tools/__tasks__/compile/javascript/rjs--webpack.js [hunk 1]
```diff
@@ -148,6 +148,16 @@ const bundles = [{
         'inlineSvg',
     ],
 }, {
+    name: 'bootstraps/enhanced/newsletters',
```
Comment (gtrufitt) [inline tools/__tasks__/compile/javascript/rjs--webpack.js]: You'll also need this in `rjs.js` in the same folder

**Review by gtrufitt — APPROVED** [review]
LGTM - maybe @SiAdcock could have a quick 👀 to check the webpack side?

**Review by SiAdcock — APPROVED** [review]
Real good, just @gtrufitt's comment and you're golden

**Comment by katebee** [comment]
This failed to deploy on CODE... error finding file `newsletters-rjs.js`

**Pull request merged**